py-glaredb/src/session.rs

```diff
@@ -118,7 +130,7 @@ impl PyExecutionResult {
     /// Convert to Arrow Table
     /// Collect the batches and pass to Arrow Table
     #[allow(clippy::wrong_self_convention)] // this is consistent with other python API's
-    fn to_arrow(&mut self, py: Python) -> PyResult<PyObject> {
+    pub fn to_arrow(&mut self, py: Python) -> PyResult<PyObject> {
```

Comment:
Does `pub` do anything special when using the `pymethods` macro?

Reply:
i don't think it has any impact on the python bindings. I used `pub` because i need to access from the other struct `PyLogicalPlan`. I suppose i could do `pub(crate)` as well.

Reply:
I'm down keeping it `pub`.